src/zc/buildout/easy_install.py

```diff
@@ -399,11 +400,11 @@ def _call_easy_install(self, spec, ws, dest, dist):
             for d in dists:
                 newloc = os.path.join(dest, os.path.basename(d.location))
                 if os.path.exists(newloc):
-                    if os.path.isdir(newloc):
-                        shutil.rmtree(newloc)
-                    else:
-                        os.remove(newloc)
-                os.rename(d.location, newloc)
+                    logger.warn(
+                        "Path %s already exists.\nIf this contains a wrong "
+                        "package, please remove it yourself.", newloc)
+                else:
+                    os.rename(d.location, newloc)
```

Comment:
See also discussion at #307 . I am updating and amending the pull request to reflect this discussion.